packages/medulas-react-components/src/utils/storybook/index.tsx

```diff
@@ -1,10 +1,31 @@
 import ThemeProvider from '../../theme/MedulasThemeProvider';
+import { Provider } from 'react-redux';
+import { createMemoryHistory, History } from 'history';
+import { ConnectedRouter, connectRouter } from 'connected-react-router';
+import { createStore, combineReducers } from 'redux';
 import * as React from 'react';
 
+const history = createMemoryHistory();
+
+const createRootReducer = (history: History) =>
+  combineReducers({
+    router: connectRouter(history),
+  });
+
+const store = createStore(createRootReducer(history));
+
 interface Props {
   readonly children: React.ReactNode;
 }
 
 export const Storybook = ({ children }: Props): JSX.Element => {
-  return <ThemeProvider injectFonts>{children}</ThemeProvider>;
+  return (
```

Comment:
The `medulas-react-components` package should not expose any redux or react router or history logic. Why do you need it?

Reply:
Because <Link> in storybook will not work otherwise.